login.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+	session_start();
+	
+	header('Content-type: application/json');
+	
+	$inData = getRequestInfo();
+	
+	$id = 0;
+	$firstName = "";
+	$lastName = "";
+	
+	$database_url = parse_url(getenv('DATABASE_URL'));
+
+	$server = $database_url['host'];
+	$user = $database_url['user'];
+	$pass = $database_url['pass'];
+	$db = substr($database_url['path'], 1);
+	
+	$conn = new mysqli($server, $user, $pass, $db);
+	if($conn->connect_error)
+	{
+		returnWithError($conn->connect_error);
+	}
+	else
+	{
+		$stmt = $conn->prepare("SELECT ID,firstName,lastName FROM Users WHERE Login=? AND Password =?");
```

Comment:
This one is my fault, I didn't give you the ERD until now. These fields should be using the ones specified in the ERD for our database. They are also case sensitive.